src/services/multiversx-communication/mx.api.service.ts

```diff
@@ -235,23 +256,14 @@ export class MXApiService {
 
     async getNftsForUser(
         address: string,
-        from = 0,
-        size = 100,
         type = 'MetaESDT',
-        collections?: string[],
     ): Promise<NftToken[]> {
         const nfts: NftToken[] = await this.genericGetExecutor.execute({
             methodName: this.getNftsForUser.name,
             resourceUrl: `accounts/${address}/nfts?type=${type}&size=${constantsConfig.MAX_USER_NFTS}&fields=identifier,collection,ticker,decimals,timestamp,attributes,nonce,type,name,creator,royalties,uris,url,tags,balance,assets`,
         });
 
-        const userNfts = collections
-            ? nfts
-                  .filter((nft) => collections.includes(nft.collection))
-                  .slice(from, size)
-            : nfts.slice(from, size);
-
-        for (const nft of userNfts) {
+        for (const nft of nfts) {
```

Comment:
why did we remove the filter?

Reply:
i see you are doing the filter based on the creator now, but maybe it would be better to do the filter here

Reply:
filtering is now done at the context getter level